LanguageSkills/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
     "LanguageSkillsDBConnection": "Server=127.0.0.1;;Port=5432;Database=LanguageSkillsDB;Username=postgres;Password=1111"
+    //Todo Delete the comment
```

Comment:
Please pay attention that only one connection string should be uncommented at specific period of time.